flagmap.go

```diff
@@ -64,6 +64,23 @@ func (f FlagMap) String(long string) string {
 	return s
 }
 
+// StringSlice returns the given flag value as string slice.
+// Panics if not present. Flags must be registered.
+func (f FlagMap) StringSlice(long string) []string {
+	i := f[long]
+	if i == nil {
+		panic(fmt.Errorf("missing flag value: flag '%s' not registered", long))
+	}
+	s := make([]string, 0)
+	if len(i.Value.([]interface{})) == 0 {
+		return s
+	}
+	for _, v := range i.Value.([]interface{}) {
```

Comment:
when the slice above is initialized to its proper length already, use the loop index here to directly assign the value, avoiding `append` completely

Reply:
done